app/ui/src/main/java/com/fsck/k9/ui/K9Drawer.kt

```diff
@@ -174,6 +175,12 @@ class K9Drawer(private val parent: MessageList, savedInstanceState: Bundle?) : K
             selectUnifiedInbox()
         } else {
             unifiedInboxSelected = false
+            accentColor = if (parent.themeManager.appTheme == Theme.DARK) {
```

Comment:
Please don't use `parent` to retrieve dependencies. If you need a dependency add them using Koin (`by inject()` for now, see top of the class).